app/src/main/res/layout/activity_numeric_input_interaction_view_test.xml

```diff
@@ -34,18 +35,34 @@
     android:maxLength="200"
     android:padding="8dp"
     android:singleLine="true" />
+
   <org.oppia.app.customview.interaction.FractionInputInteractionView
     android:id="@+id/test_fraction_input_interaction_view"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:layout_margin="8dp"
     android:background="@drawable/edit_text_background"
-    android:focusable="true"
     android:digits="0123456789/- "
+    android:focusable="true"
     android:hint="Write fraction here."
     android:inputType="number"
     android:longClickable="false"
     android:maxLength="200"
     android:padding="8dp"
     android:singleLine="true" />
+
+  <org.oppia.app.customview.interaction.NumberWithUnitsInputInteractionView
+    android:id="@+id/test_number_with_units_input_interaction_view"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="8dp"
+    android:background="@drawable/edit_text_background"
+    android:digits="0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ $.-/*^()"
```

Comment:
This input type isn't numbers, so I don't think this filter is actually used. We'll need to implement this manually in the custom view as part of validation.